site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -778,6 +778,30 @@ components:
                description: The IDs of the gateway groups that this device is a member of. Note that "via" and "memberOf" must not be set at the same time. The same applies for "viaGroups" and "memberOf" which must be set at the same time too. The reason is that Eclipse Hono does not support groups of gateway groups.
             "ext":
                $ref: '#/components/schemas/Extensions'
+            "status":
+               $ref: '#/components/schemas/Status'
```

Comment:
There are several changes to the API queued at the moment, so I propose to do a version bump when all of the changes are agreed ? there are a couple of other operations I will add in the next few days :) 
I'll do another PR with the version change and we'll merge all all of those at the same time. WDYT ?

Reply:
I'd rather increase the version number with each PR so that we do not run at risk of losing track of it. That way, the PRs are more self contained as well and we can decide on each one of them what type of version change it constitutes ...